frontend/packages/core/src/chip.tsx

```diff
@@ -86,6 +86,7 @@ const StyledChip = styled(MuiChip)<{
     },
   },
   props => ({
+    height: props.size === "small" ? "24px" : "32px",
```

Comment:
Is this reflected in the design system (figma)?

Reply:
Yeah, the figma has the chip as 24px and our default was 32 so I made the small == 24